src/validation.cpp

```diff
@@ -3461,13 +3448,13 @@ static bool ContextualCheckBlockHeader(const CBlockHeader& block, BlockValidatio
  *  in ConnectBlock().
  *  Note that -reindex-chainstate skips the validation that happens here!
  */
-static bool ContextualCheckBlock(const CBlock& block, BlockValidationState& state, const Consensus::Params& consensusParams, const CBlockIndex* pindexPrev)
+static bool ContextualCheckBlock(const CBlock& block, BlockValidationState& state, const Consensus::Params& consensusParams, const CBlockIndex* pindexPrev) EXCLUSIVE_LOCKS_REQUIRED(cs_main)
```

Comment:
In commit "[refactor] Add deploymentstatus.h":

Is this "EXCLUSIVE_LOCKS_REQUIRED(cs_main)" still needed?

Reply:
No, don't think so. Removed.